Cargo.toml

```diff
@@ -9,14 +9,19 @@ repository = "https://github.com/gfx-rs/naga"
 keywords = ["shader", "SPIR-V"]
 license = "MPL-2.0"
 
+[features]
+serialize = ["serde", "spirv/serialize"]
+deserialize = ["serde", "spirv/deserialize"]
+
 [dependencies]
 bitflags = "1"
 fxhash = "0.2"
 log = "0.4"
 num-traits = "0.2"
+serde = { version = "1", optional = true, features = ["derive"] }
 spirv = { package = "spirv_headers", version = "1" }
 
 [dev-dependencies]
 env_logger = "0.6"
 ron = "0.5"
-serde = { version = "1", features = ["serde_derive"] }
+serde = { version = "1", features = ["derive"] }
```

Comment:
this feature doesn't exist in 1.0.0: https://crates.io/crates/serde/1.0.0
let's use `serde_derive` for compatibility, or put a specific serde version here that has that

Reply:
Got it; I went with the `serde_derive` feature